### ISSUE TITLE
Error: Keyring should be initialised via 'loadAll' before use

### ISSUE THREAD

**Issue by matt-paul** [substrate-developer-hub/substrate-front-end-template#219]
Hello. I have based my project off this repo, specifically making use of  'useSubstrate'. 

In a component in my app, i have the following...

```
 const { api, apiState, keyring, keyringState } = useSubstrate();

 if (keyringState === "READY" && apiState === "READY") {
  
    const res = keyring && keyring.getPairs();
    ...etc....
   
  }
```

I receive the following error:
<img width="1077" alt="Screenshot 2021-12-18 at 19 16 39" src="https://user-images.githubusercontent.com/13286587/146653222-6ffe306e-73b4-41b3-91b0-797c4923c288.png">

Could you help understand where else i could look to track down the issue, if both `keyringState` and `apiState` are in `READY` state?

**Comment by jimmychu0807** [comment]
@matt-paul 

`useSubstrate()` depends on `src/substrate-lib/SubstrateContext.js`, and your outermost `<Main>` component need to be surrounded by `<SubstrateContextProvider>`, [as in here](https://github.com/substrate-developer-hub/substrate-front-end-template/blob/main/src/App.js#L87-L89). 

Did you do both of them?

**Comment by hoangdv1995** [comment]
@jimmychu0807 I got the same error when React.StrictMode is enabled

**Comment by jimmychu0807** [comment]
@hoangdv1995 @matt-paul 
Thanks for reporting this issue, and especially enabling the React strict mode, which is about React doing double rendering. I can reproduce the issue and is fixed in #227.
